src/app/application/index.ts

```diff
@@ -56,6 +56,7 @@ declare module '@vue/runtime-core' {
 }
 
 const $ = {
+  ...ME,
```

Comment:
We use/expose `ME` dependency injection tokens so we can overwrite them elsewhere.